populate_db.py

```diff
@@ -57,6 +58,10 @@ def create_settings():
     get_or_create(Setting, app_name='Open Event')
 
 
+def create_modules():
+    get_or_create(Module, donation_include=False)
```

Comment:
Is this required? I think all values are by default false in model itself.

Reply:
So should it be `get_or_create(Module)`